web/src/styles/operator.css

```
@@ -0,0 +1,595 @@
+  color: var(--text-tertiary);
+}
+
+.op-eyebrow-strong {
+  color: var(--text-secondary);
+}
+
+/* ───────────────────────── Status LED + label */
+
+.op-led {
+  --led-color: var(--text-tertiary);
+  display: inline-block;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)